Plugin.pm

```diff
@@ -288,6 +288,34 @@ sub trackInfoMenu {
 
 	my $search = cstring($client, 'SEARCH');
 	my $items = [];
+
+	my $artists = $track->remote ? $remoteMeta->{artists} : undef;
+	my $album_id = $track->remote ? $remoteMeta->{album_details}->{id} : undef;
+	my $track_id = Plugins::TIDAL::ProtocolHandler::_getId($track->url);
+
+	push @$items, {
+		name => $album,
+		line1 => $album,
+		line2 => $artist,
+		favorites_url => 'tidal://album:' . $album_id,
+		type => 'playlist',
+		url => \&getAlbum,
+		image => Plugins::TIDAL::API->getImageUrl($remoteMeta, 'usePlaceholder'),
```

Comment:
Unless the majority of the items I that list have an icon, I'd rather not put an icon here. It could lead to grid display, which is ugly when there's only one or two menu items with an icon among many with the default image.

Reply:
Perfect!